### ISSUE TITLE
Prefill security API key

### ISSUE THREAD

**Issue by sebj54** [mrin9/RapiDoc#72]
Hi!

Thank you for your awesome web component, it does its job very well!

I just encountered a bug (I guess). I have an API which has a specific HTTP header, here is the definition of the security scheme (extract of my openapi.json):

```json
{
    "securitySchemes": {
        "bearerAuth": {
            "type": "apiKey",
            "in": "header",
            "name": "X-Authorization",
            "description": "Value format: \"Bearer xxxxx\" where xxxxx is replaced by a valid JWT token issued from Firebase. See: https://firebase.google.com/docs/auth"
        }
    }
}
```

I would have liked to prefill the API key field:

![image](https://user-images.githubusercontent.com/5880435/67789665-a2715f00-fa74-11e9-8a50-71bdcea59aa8.png)

But it seems that it cannot be set, even if I use the `api-key-value` attribute on the `<rapi-doc>` element.

Thanks for your help!

**Comment by mrin9** [comment]
thanks for pointing out that regression. I will get it fixed soon

**Comment by mrin9** [comment]
this is now been fixed in release `7.0.0`
below 3 attributes can be used to make it work
- `api-key-name`
- `api-key-value`
- `api-key-location`

feel free to reopen in case it is not working as you were expecting it to be

**Comment by sebj54** [comment]
@mrin9 It works fine! Thanks for the quick fix 👍
